api/proto/disperser/v2/disperser_v2.proto

```diff
@@ -29,6 +29,8 @@ message DisperseBlobRequest {
   // The blob to be dispersed.
   //
   // The size of this byte array may be any size as long as it does not exceed the maximum length of 16MiB.
+  // While there is no minimum size, the blob size charged against the payment method will be rounded up to
```

Comment:
Can it be 0? len(blob)>0 is required right?

Reply:
you are right, fixed the sentence